app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -19,13 +19,14 @@ talk_to_us: false
 inset_text:
   applications-open:
     text: |-
-      <p>Applications for the international relocation payment (IRP) are now closed. You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen in autumn 2024.</p>
 
       <h3>Changes to eligibility and payment</h3>
```

Comment:
Page starts with an h3 which is odd? But this is in the inset text - I'm not sure we can use headings here in this way?

Reply:
I wasn't sure if it was something to do with it being in the inset box?

Reply:
It looks like headings work in the inset box

However, we can't jump from an h1 to an h3 - that is an accessibility fail

I've tried restyling as an h2 and it doesn't look good - the h2 is styled smaller than then h3 in this inset component
![image](https://github.com/user-attachments/assets/dca5a277-44ec-4421-b754-ce37a5706e8d)

I think we need to find a pragmatic solution that enables us to get this out but making sure it is accessible?

Reply:
Do we need the top current h2? Then we could have the two existing h3s as h2s?

Reply:
good idea!

Reply:
you will need to add another h2 below the inset box either way otherwise this info would sit until the h2 in the inset box?

![image](https://github.com/user-attachments/assets/84af0528-0373-4528-84bc-d799ff99ee25)